.github/workflows/reusable-publish-images.yaml

```diff
@@ -38,28 +43,20 @@ jobs:
             type=ref,event=branch
 
       - name: Set up QEMU
-        uses: docker/setup-qemu-action@v3
+        uses: docker/setup-qemu-action@29109295f81e9208d7d86ff1c6c12d2833863392 # v3.6.0
 
       - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v3
-
-      - name: Cache Docker layers
```

Comment:
Why was this step removed, is this caching step a security issue?

Reply:
these steps were flagged as https://woodruffw.github.io/zizmor/audits/#cache-poisoning issue.

Reply:
Ok, I've no objections to removing the cache step in the publish workflow.

Could you also remove the references to `/tmp/.buildx-cache` in the same workflow? (`cache-from` and `cache-to` of the `docker/build-push-action` action, and a workaround to move the cache)